src/nvim/event/process.c

```diff
@@ -126,7 +126,6 @@ void process_teardown(Loop *loop) FUNC_ATTR_NONNULL_ALL
       CREATE_EVENT(loop->events, process_close_handles, 1, proc);
     } else {
       uv_kill(proc->pid, SIGTERM);
```

Comment:
This could also be removed, because `process_stop()` sends signal `SIGTERM` without waiting.